### PR TITLE
dmapd: update to 0.0.84

### PR DESCRIPTION
**Pull request by flyn-org**
Signed-off-by: W. Michael Petullo <mike@flyn.org>

Maintainer: me
Compile tested: x86_64 master
Run tested: x86_64 master

Description:

### CODE REVIEW

File: net/dmapd/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.flyn.org/projects/dmapd
-PKG_HASH:=ab372ff217d2a74a33d62098fe501da663e5dded0df67e2c2c83d968fa9ac7b9
+PKG_HASH:=96d134e7c6d8e0394ee654903a184bfc200a3139692d124e7b11a3400138c8b2
 
 PKG_FIXUP:=autoreconf
```
Comment (neheb) [inline net/dmapd/Makefile]: I don't think this is needed.

Comment (neheb) [reply net/dmapd/Makefile]: ping

Comment (flyn-org) [reply net/dmapd/Makefile]: I do not understand. 96d1... is the SHA-256 hash of 0.0.84.

Comment (neheb) [reply net/dmapd/Makefile]: The autoreconf line. Nothing is being patched.

**Comment by flyn-org** [comment]
Okay. Makes sense. I was looking at the +/-.

**Pull request merged**